contrib/qat/private_key_providers/source/qat.h

```diff
@@ -93,8 +93,11 @@ class QatManager : public std::enable_shared_from_this<QatManager>,
   static int connectionIndex();
   static int contextIndex();
 
+  bool checkQatDevice();
+
 private:
   LibQatCryptoSharedPtr libqat_{};
+  bool qat_is_supported{true};
```

Comment:
nit: `qat_is_supported_`.

Our clang tidy is still be disabled, or the ci will find this.

Reply:
ok, done, thanks!